examples/14. Advanced Authentication.md

```diff
@@ -0,0 +1,64 @@
+FORMAT: 1A
+
+# Advanced Authentication API
+
+## API Blueprint
++ [Previous: Authentication](13.%20Authentication.md)
++ [This: Raw API Blueprint](https://raw.github.com/apiaryio/api-blueprint/master/examples/14.%20Advanced%20Authentication.md)
+
+## Authentication
+The API requires OAuth2 access token for authentication. Available scopes for OAuth2 are `user:read`, `user:write` and `email`.
+
++ Authentication Schemes
+    + oauth (OAuth2 Authentication Scheme) - Using the oauth2 authentication framework
+        + scopes: `user:read`, `user:write`, email
+
++ Response 403 (application/json)
+
+        {
+          "error": "Requires Authentication"
+        }
+
+## User [/user/{id}]
+
++ Parameters
+    + id (string) - Username of the user
+
+### Retrieve a User Profile [GET]
+This endpoint returns an error if no authentication (access token) is given and returns only the username of an user unless the oauth scope of the access token is `user:write` or `email`.
+
++ Authenticated: oauth[`user:read`]
+
++ Response 200 (application/json)
+
+        {
+          "username": "john"
+        }
+
++ Request (application/json)
+    + Authenticated: oauth[`user:write`, email]
+
++ Response 200 (application/json)
+
+        {
+          "username": "john",
+          "email": "john@appleseed.com"
+        }
+
+### Update a User Profile [POST]
+This endpoint allows the user to update an user profile only if the oauth scope of the access token is `user:write`.
+
++ Authenticated: oauth[`user:write`]
```

Comment:
Is this a sort of escaping? it wasn't mentioned anywhere in the spec above, not even in a side note. In case it's blueprint-wide way to escape things, I think scopes with `:` in them are so common that this needs to be at least in a note.